### ISSUE TITLE
'PUSH_NOTIFICATION_REDACT_CONTENT' can not be specified in docker-compose file

### ISSUE THREAD

**Issue by R4WBIT** [zulip/docker-zulip#203]
It seems like the variable `PUSH_NOTIFICATION_REDACT_CONTENT: 'True'` does not have any effect on the server when specified in the `docker-compose.yml` file.

Can this be included, or is there another preferred way to do it?  
Would rather not mount the settings-file to the container.

**Comment by sirpengi** [comment]
It should be:
```
environment:
  SETTING_PUSH_NOTIFICATION_REDACT_CONTENT: "True"
```

(that is, you're missing the `SETTING_` prefix)

**Comment by timabbott** [comment]
Yep, that's correct -- we document you need to use the `SETTING_` prefix for anything that you want in `settings.py`.  

We should probably figure out a way to do some sort of warning for this type of situation, if it's possible.
